SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp

```diff
@@ -58,11 +58,9 @@ void FinishSetupConnectionConfig(USpatialConnectionManager* ConnectionManager, c
 		FReceptionistConfig& ReceptionistConfig = ConnectionManager->ReceptionistConfig;
 		ReceptionistConfig.WorkerType = WorkerType;
 
-		const TCHAR* UseExternalIpForBridge = TEXT("useExternalIpForBridge");
-		if (URL.HasOption(UseExternalIpForBridge))
+		if (URL.HasOption(*SpatialConstants::URL_USE_EXTERNAL_IP_FOR_BRIDGE_OPTION))
```

Comment:
Should we have to do this?

Reply:
Yeah don't think we have to do this anymore

Reply:
Might also be appropriate to change in StartSetupConnectionConfigFromURL:

ConnectionManager->ReceptionistConfig.SetReceptionistHost(URL.Host);  -> ConnectionManager->ReceptionistConfig.SetReceptionistHost(URL);